src/gui/properties/trackerlist.cpp

```diff
@@ -112,10 +112,13 @@ TrackerList::TrackerList(PropertiesWidget *properties)
     headerItem()->setTextAlignment(COL_PEERS, (Qt::AlignRight | Qt::AlignVCenter));
     headerItem()->setTextAlignment(COL_DOWNLOADED, (Qt::AlignRight | Qt::AlignVCenter));
     // Set hotkeys
-    m_editHotkey = new QShortcut(Qt::Key_F2, this, SLOT(editSelectedTracker()), 0, Qt::WidgetShortcut);
-    connect(this, SIGNAL(doubleClicked(QModelIndex)), SLOT(editSelectedTracker()));
-    m_deleteHotkey = new QShortcut(QKeySequence::Delete, this, SLOT(deleteSelectedTrackers()), 0, Qt::WidgetShortcut);
-    m_copyHotkey = new QShortcut(QKeySequence::Copy, this, SLOT(copyTrackerUrl()), 0, Qt::WidgetShortcut);
+    m_editHotkey = new QShortcut(Qt::Key_F2, this, 0, 0, Qt::WidgetShortcut);
```

Comment:
you might want to use `nullptr` here and other similar instances.

Reply:
I'll rebase the PR that is specifically about these or this PR depending which gets merged first.